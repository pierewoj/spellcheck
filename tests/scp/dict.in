insert item
insert scp
insert object
insert class
insert safe
insert special
insert containment
insert procedures
insert scp
insert is
insert to
insert be
insert kept
insert inside
insert a
insert secure
insert video
insert storage
insert unit
insert at
insert the
insert media
insert archive
insert of
insert site
insert when
insert in
insert use
insert scp
insert should
insert not
insert be
insert removed
insert from
insert its
insert casing
insert or
insert exposed
insert to
insert any
insert strong
insert magnetic
insert sources
insert a
insert betamax
insert home
insert video
insert system
insert and
insert an
insert analog
insert television
insert has
insert been
insert provided
insert in
insert observation
insert theatre
insert at
insert site
insert as
insert well
insert as
insert video
insert equipment
insert to
insert record
insert viewings
insert description
insert scp
insert is
insert a
insert standard
insert betamax
insert tape
insert ronald
insert regan
insert cut
insert up
insert while
insert talking
insert sic
insert has
insert been
insert handwritten
insert on
insert the
insert adhesive
insert sticker
insert in
insert felt
insert tip
insert pen
insert laboratory
insert analysis
insert indicates
insert that
insert scp
insert is
insert made
insert of
insert ordinary
insert material
insert and
insert serial
insert numbers
insert correspond
insert with
insert home
insert cassette
insert tapes
insert produced
insert in
insert september
insert of
insert scp
insert was
insert initially
insert encountered
insert by
insert a
insert filing
insert clerk
insert in
insert the
insert ronald
insert reagan
insert presidential
insert library
insert in
insert who
insert upon
insert watching
insert it
insert alerted
insert the
insert police
insert with
insert the
insert intent
insert to
insert find
insert the
insert tapes
insert creator
insert to
insert press
insert obscenity
insert charges
insert a
insert low
insert level
insert police
insert investigation
insert was
insert conducted
insert at
insert which
insert point
insert the
insert foundation
insert was
insert alerted
insert and
insert secured
insert scp
insert class
insert a
insert amnesiacs
insert were
insert administered
insert before
insert could
insert be
insert notified
insert further
insert investigation
insert of
insert the
insert librarys
insert records
insert by
insert foundation
insert personnel
insert failed
insert to
insert yield
insert any
insert leads
insert on
insert scps
insert origin
insert scp
insert appears
insert to
insert be
insert a
insert home
insert video
insert recording
insert of
insert former
insert united
insert states
insert president
insert ronald
insert reagan
insert delivering
insert his
insert evil
insert empire
insert speech
insert to
insert the
insert national
insert association
insert of
insert evangelicals
insert at
insert sheraton
insert twin
insert towers
insert hotel
insert orlando
insert fl
insert on
insert however
insert at
insert minute
insert and
insert seconds
insert the
insert speech
insert begins
insert to
insert deviate
insert heavily
insert eventually
insert resembling
insert no
insert known
insert speech
insert ever
insert made
insert by
insert reagan
insert beginning
insert at
insert approximately
insert minutes
insert multiple
insert incisions
insert lacerations
insert and
insert penetration
insert wounds
insert can
insert be
insert seen
insert being
insert slowly
insert inflicted
insert though
insert no
insert corresponding
insert source
insert of
insert these
insert wounds
insert is
insert visible
insert despite
insert suffering
insert bodily
insert harm
insert that
insert would
insert likely
insert incapacitate
insert an
insert ordinary
insert person
insert reagan
insert will
insert continue
insert to
insert deliver
insert his
insert speech
insert until
insert either
insert his
insert vocal
insert cords
insert are
insert severed
insert or
insert the
insert tape
insert degrades
insert to
insert static
insert at
insert upon
insert rewinding
insert scp
insert and
insert initiating
insert playback
insert reagan
insert will
insert deliver
insert an
insert entirely
insert new
insert speech
insert often
insert radically
insert different
insert from
insert the
insert ones
insert previously
insert observed
insert topics
insert have
insert included
insert torture
insert child
insert molestation
insert and
insert ritual
insert sacrifice
insert trauma
insert inflicted
insert upon
insert reagan
insert also
insert appears
insert to
insert be
insert divergent
insert with
insert impalement
insert genital
insert mutilation
insert and
insert redacted
insert having
insert all
insert been
insert observed
insert in
insert roughly
insert one
insert in
insert seven
insert viewings
insert of
insert scp
insert a
insert figure
insert clothed
insert in
insert black
insert robes
insert with
insert a
insert conical
insert hood
insert will
insert have
insert replaced
insert a
insert random
insert member
insert of
insert reagans
insert press
insert detail
insert henceforth
insert referred
insert to
insert as
insert scp
insert the
insert significance
insert of
insert the
insert appearance
insert of
insert scp
insert is
insert currently
insert unknown
insert the
insert speeches
insert delivered
insert by
insert reagan
insert are
insert mostly
insert incoherent
insert lacking
insert any
insert sort
insert of
insert underlying
insert thematic
insert structure
insert and
insert largely
insert being
insert composed
insert of
insert nonsensical
insert anecdotes
insert and
insert parables
insert however
insert occasionally
insert references
insert are
insert made
insert to
insert future
insert events
insert that
insert reagan
insert could
insert not
insert possibly
insert have
insert known
insert about
insert or
insert predicted
insert such
insert as
insert the
insert september
insert terrorist
insert attacks
insert the
insert result
insert of
insert the
insert russian
insert elections
insert and
insert for
insert this
insert reason
insert rigorous
insert time
insert and
insert effort
insert has
insert been
insert devoted
insert to
insert recording
insert the
insert speech
insert delivered
insert on
insert each
insert playback
insert attempts
insert to
insert replicate
insert scp
insert onto
insert a
insert similar
insert betamax
insert tape
insert have
insert met
insert with
insert failure
insert however
insert cameras
insert used
insert to
insert record
insert the
insert television
insert scp
insert is
insert broadcasted
insert on
insert have
insert succeeded
insert in
insert capturing
insert individual
insert playbacks
insert any
insert observations
insert performed
insert on
insert scp
insert must
insert be
insert recorded
insert on
insert the
insert camcorder
insert provided
insert and
insert delivered
insert for
insert subsequent
insert review
insert to
insert dr
insert project
insert supervisor
insert years
insert of
insert natural
insert magnetic
insert interference
insert has
insert severely
insert degraded
insert scps
insert signal
insert quality
insert making
insert it
insert even
insert more
insert difficult
insert to
insert sift
insert meaningful
insert information
insert from
insert playbacks
insert additionally
insert the
insert gruesome
insert nature
insert of
insert the
insert mutilations
insert performed
insert upon
insert reagan
insert have
insert been
insert described
insert as
insert extremely
insert disturbing
insert and
insert for
insert this
insert reason
insert it
insert is
insert recommended
insert that
insert any
insert personnel
insert feeling
insert squeamish
insert or
insert ill
insert after
insert playback
insert visit
insert the
insert on
insert site
insert psychiatry
insert facility
insert for
insert a
insert level
insert evaluation
insert as
insert ronald
insert reagan
insert was
insert alive
insert at
insert the
insert time
insert of
insert scps
insert containment
insert a
insert surveillance
insert net
insert was
insert deployed
insert to
insert establish
insert any
insert relation
insert between
insert him
insert and
insert scp
insert no
insert known
insert connection
insert was
insert developed
insert though
insert reagan
insert would
insert frequently
insert complain
insert about
insert nightmares
insert before
insert his
insert mental
insert state
insert degenerated
insert due
insert to
insert alzheimers
insert excerpt
insert from
insert video
insert transcript
insert of
insert recording
insert made
insert on
insert excerpt
insert from
insert video
insert transcript
insert of
insert recording
insert made
insert on
insert video
insert transcript
insert of
insert recording
insert made
insert on
insert video
insert transcript
insert of
insert recording
insert made
insert on
save slownik.dict

