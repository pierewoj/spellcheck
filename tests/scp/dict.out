inserted: item
inserted: scp
inserted: object
inserted: class
inserted: safe
inserted: special
inserted: containment
inserted: procedures
ignored
inserted: is
inserted: to
inserted: be
inserted: kept
inserted: inside
inserted: a
inserted: secure
inserted: video
inserted: storage
inserted: unit
inserted: at
inserted: the
inserted: media
inserted: archive
inserted: of
inserted: site
inserted: when
inserted: in
inserted: use
ignored
inserted: should
inserted: not
ignored
inserted: removed
inserted: from
inserted: its
inserted: casing
inserted: or
inserted: exposed
ignored
inserted: any
inserted: strong
inserted: magnetic
inserted: sources
ignored
inserted: betamax
inserted: home
ignored
inserted: system
inserted: and
inserted: an
inserted: analog
inserted: television
inserted: has
inserted: been
inserted: provided
ignored
inserted: observation
inserted: theatre
ignored
ignored
inserted: as
inserted: well
ignored
ignored
inserted: equipment
ignored
inserted: record
inserted: viewings
inserted: description
ignored
ignored
ignored
inserted: standard
ignored
inserted: tape
inserted: ronald
inserted: regan
inserted: cut
inserted: up
inserted: while
inserted: talking
inserted: sic
ignored
ignored
inserted: handwritten
inserted: on
ignored
inserted: adhesive
inserted: sticker
ignored
inserted: felt
inserted: tip
inserted: pen
inserted: laboratory
inserted: analysis
inserted: indicates
inserted: that
ignored
ignored
inserted: made
ignored
inserted: ordinary
inserted: material
ignored
inserted: serial
inserted: numbers
inserted: correspond
inserted: with
ignored
inserted: cassette
inserted: tapes
inserted: produced
ignored
inserted: september
ignored
ignored
inserted: was
inserted: initially
inserted: encountered
inserted: by
ignored
inserted: filing
inserted: clerk
ignored
ignored
ignored
inserted: reagan
inserted: presidential
inserted: library
ignored
inserted: who
inserted: upon
inserted: watching
inserted: it
inserted: alerted
ignored
inserted: police
ignored
ignored
inserted: intent
ignored
inserted: find
ignored
ignored
inserted: creator
ignored
inserted: press
inserted: obscenity
inserted: charges
ignored
inserted: low
inserted: level
ignored
inserted: investigation
ignored
inserted: conducted
ignored
inserted: which
inserted: point
ignored
inserted: foundation
ignored
ignored
ignored
inserted: secured
ignored
ignored
ignored
inserted: amnesiacs
inserted: were
inserted: administered
inserted: before
inserted: could
ignored
inserted: notified
inserted: further
ignored
ignored
ignored
inserted: librarys
inserted: records
ignored
ignored
inserted: personnel
inserted: failed
ignored
inserted: yield
ignored
inserted: leads
ignored
inserted: scps
inserted: origin
ignored
inserted: appears
ignored
ignored
ignored
ignored
ignored
inserted: recording
ignored
inserted: former
inserted: united
inserted: states
inserted: president
ignored
ignored
inserted: delivering
inserted: his
inserted: evil
inserted: empire
inserted: speech
ignored
ignored
inserted: national
inserted: association
ignored
inserted: evangelicals
ignored
inserted: sheraton
inserted: twin
inserted: towers
inserted: hotel
inserted: orlando
inserted: fl
ignored
inserted: however
ignored
inserted: minute
ignored
inserted: seconds
ignored
ignored
inserted: begins
ignored
inserted: deviate
inserted: heavily
inserted: eventually
inserted: resembling
inserted: no
inserted: known
ignored
inserted: ever
ignored
ignored
ignored
inserted: beginning
ignored
inserted: approximately
inserted: minutes
inserted: multiple
inserted: incisions
inserted: lacerations
ignored
inserted: penetration
inserted: wounds
inserted: can
ignored
inserted: seen
inserted: being
inserted: slowly
inserted: inflicted
inserted: though
ignored
inserted: corresponding
inserted: source
ignored
inserted: these
ignored
ignored
inserted: visible
inserted: despite
inserted: suffering
inserted: bodily
inserted: harm
ignored
inserted: would
inserted: likely
inserted: incapacitate
ignored
ignored
inserted: person
ignored
inserted: will
inserted: continue
ignored
inserted: deliver
ignored
ignored
inserted: until
inserted: either
ignored
inserted: vocal
inserted: cords
inserted: are
inserted: severed
ignored
ignored
ignored
inserted: degrades
ignored
inserted: static
ignored
ignored
inserted: rewinding
ignored
ignored
inserted: initiating
inserted: playback
ignored
ignored
ignored
ignored
inserted: entirely
inserted: new
ignored
inserted: often
inserted: radically
inserted: different
ignored
ignored
inserted: ones
inserted: previously
inserted: observed
inserted: topics
inserted: have
inserted: included
inserted: torture
inserted: child
inserted: molestation
ignored
inserted: ritual
inserted: sacrifice
inserted: trauma
ignored
ignored
ignored
inserted: also
ignored
ignored
ignored
inserted: divergent
ignored
inserted: impalement
inserted: genital
inserted: mutilation
ignored
inserted: redacted
inserted: having
inserted: all
ignored
ignored
ignored
inserted: roughly
inserted: one
ignored
inserted: seven
ignored
ignored
ignored
ignored
inserted: figure
inserted: clothed
ignored
inserted: black
inserted: robes
ignored
ignored
inserted: conical
inserted: hood
ignored
ignored
inserted: replaced
ignored
inserted: random
inserted: member
ignored
inserted: reagans
ignored
inserted: detail
inserted: henceforth
inserted: referred
ignored
ignored
ignored
ignored
inserted: significance
ignored
ignored
inserted: appearance
ignored
ignored
ignored
inserted: currently
inserted: unknown
ignored
inserted: speeches
inserted: delivered
ignored
ignored
ignored
inserted: mostly
inserted: incoherent
inserted: lacking
ignored
inserted: sort
ignored
inserted: underlying
inserted: thematic
inserted: structure
ignored
inserted: largely
ignored
inserted: composed
ignored
inserted: nonsensical
inserted: anecdotes
ignored
inserted: parables
ignored
inserted: occasionally
inserted: references
ignored
ignored
ignored
inserted: future
inserted: events
ignored
ignored
ignored
ignored
inserted: possibly
ignored
ignored
inserted: about
ignored
inserted: predicted
inserted: such
ignored
ignored
ignored
inserted: terrorist
inserted: attacks
ignored
inserted: result
ignored
ignored
inserted: russian
inserted: elections
ignored
inserted: for
inserted: this
inserted: reason
inserted: rigorous
inserted: time
ignored
inserted: effort
ignored
ignored
inserted: devoted
ignored
ignored
ignored
ignored
ignored
ignored
inserted: each
ignored
inserted: attempts
ignored
inserted: replicate
ignored
inserted: onto
ignored
inserted: similar
ignored
ignored
ignored
inserted: met
ignored
inserted: failure
ignored
inserted: cameras
inserted: used
ignored
ignored
ignored
ignored
ignored
ignored
inserted: broadcasted
ignored
ignored
inserted: succeeded
ignored
inserted: capturing
inserted: individual
inserted: playbacks
ignored
inserted: observations
inserted: performed
ignored
ignored
inserted: must
ignored
inserted: recorded
ignored
ignored
inserted: camcorder
ignored
ignored
ignored
ignored
inserted: subsequent
inserted: review
ignored
inserted: dr
inserted: project
inserted: supervisor
inserted: years
ignored
inserted: natural
ignored
inserted: interference
ignored
inserted: severely
inserted: degraded
ignored
inserted: signal
inserted: quality
inserted: making
ignored
inserted: even
inserted: more
inserted: difficult
ignored
inserted: sift
inserted: meaningful
inserted: information
ignored
ignored
inserted: additionally
ignored
inserted: gruesome
inserted: nature
ignored
ignored
inserted: mutilations
ignored
ignored
ignored
ignored
ignored
inserted: described
ignored
inserted: extremely
inserted: disturbing
ignored
ignored
ignored
ignored
ignored
ignored
inserted: recommended
ignored
ignored
ignored
inserted: feeling
inserted: squeamish
ignored
inserted: ill
inserted: after
ignored
inserted: visit
ignored
ignored
ignored
inserted: psychiatry
inserted: facility
ignored
ignored
ignored
inserted: evaluation
ignored
ignored
ignored
ignored
inserted: alive
ignored
ignored
ignored
ignored
ignored
ignored
ignored
inserted: surveillance
inserted: net
ignored
inserted: deployed
ignored
inserted: establish
ignored
inserted: relation
inserted: between
inserted: him
ignored
ignored
ignored
ignored
inserted: connection
ignored
inserted: developed
ignored
ignored
ignored
inserted: frequently
inserted: complain
ignored
inserted: nightmares
ignored
ignored
inserted: mental
inserted: state
inserted: degenerated
inserted: due
ignored
inserted: alzheimers
inserted: excerpt
ignored
ignored
inserted: transcript
ignored
ignored
ignored
ignored
ignored
ignored
ignored
ignored
ignored
ignored
ignored
ignored
ignored
ignored
ignored
ignored
ignored
ignored
ignored
ignored
ignored
ignored
ignored
ignored
dictionary saved in file slownik.dict
